pkg/proxy/ipvs/proxier.go

```diff
@@ -666,6 +670,26 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 		}
 	}
 
+	var nfErrCount int
```

Comment:
You don't need a count. You just need a `var foundNFConntrackModule bool`

Reply:
Dan, you're a genius (no irony here!) :) Completely forgot about using a bool.

Will make some refactoring on the function name, the function call, and make some separate commits to make it easier to read.

Thanks